crates/ruff_linter/src/rules/pylint/rules/and_or_ternary.rs

```diff
@@ -31,100 +28,15 @@ use crate::fix::snippet::SourceCodeSnippet;
 /// maximum = x if x >= y else y
 /// ```
 #[violation]
-pub struct AndOrTernary {
-    ternary: SourceCodeSnippet,
-}
+pub struct AndOrTernary;
 
+/// PLR1706
 impl Violation for AndOrTernary {
-    const FIX_AVAILABILITY: FixAvailability = FixAvailability::Sometimes;
-
-    #[derive_message_formats]
     fn message(&self) -> String {
-        if let Some(ternary) = self.ternary.full_display() {
-            format!("Consider using if-else expression (`{ternary}`)")
-        } else {
-            format!("Consider using if-else expression")
-        }
-    }
-
-    fn fix_title(&self) -> Option<String> {
-        Some(format!("Convert to if-else expression"))
```

Comment:
I remember that this needs to be a `format!` call because we extract it somewhere (or was it message). @charliermarsh do you remember the details? Do we need to preserve the `message` and `fix_title`?

Reply:
It needs to be a `format!` call for the `derive_message_formats` macro but I've hard-coded the message formats.